public/data/Jatinkumar30.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "Jatin Kumar",
+    "type": "personal",
+    "bio": "Open Source Enthusiast and Front end developer!",
+    "avatar": "https://github.com/Jatinkumar30.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/Jatinkumar30",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/JatinRawal30",
+        "icon": "twitter"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Getting grip on Open Source",
```

Comment:
@Vyvy-vi, I saw you marked as resolved my other comment. Should we change this to `open-source` or keep it as-is?

Reply:
this is a personal preference, let the author decide how they want to write it - these types of feedback should be suggestions and not blockers

I will review and merge this as part of a video I am recording today - so please do not merge, I will do this in a few hours